R/utilities.R

```diff
@@ -134,20 +132,19 @@ filter_score_prop.score_obj <- function(x, ..., prop_terms, target = NULL) {
 #'
 #' @export
 filter_score_cutoff <- function(x, ...) {
-  # TODO Rename to filter_num_terms?
   UseMethod("filter_score_cutoff")
 }
 
 #' @noRd
 #' @export
 filter_score_cutoff.score_obj <- function(x, ..., cutoff, target = NULL) {
   if (x$direction == "maximize") {
```

Comment:
We should error gracefully if a user doesn't supply the `cutoff` argument.

```suggestion
filter_score_cutoff.score_obj <- function(x, ..., cutoff, target = NULL) {
  if (rlang::is_missing(cutoff)) {
    cli::cli_abort("{.arg cutoff} must be specified")
  }
  if (x$direction == "maximize") {
```